### DISCUSSION TITLE
Type error: Cannot find module 'wagmi' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by emmajane1313** [wevm/wagmi#1210] [Q&A]
Get this error on build with Vercel and nextjs app. 

I'm using 0.7.x wagmi with latest breaking changes. Code works perfectly on local environment. Have used wagmi many times before and never experienced this issue... wondering if it has to do with the latest changes?

Full error:

```
Type error: Cannot find module 'wagmi' or its corresponding type declarations.
--
13:02:02.613 |  
13:02:02.613 | 1 \| import Image from "next/image";
13:02:02.613 | 2 \| import { FunctionComponent } from "react";
13:02:02.614 | > 3 \| import { useAccount } from "wagmi";
13:02:02.614 | \|                            ^
13:02:02.614 | 4 \| import { MetadataProps } from "../../types/general.types";
13:02:02.614 | 5 \| import useMetadata from "./hooks/useMetadata";
13:02:02.615 | 6 \|
13:02:02.654 | error Command failed with exit code 1.
13:02:02.655 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
13:02:02.678 | Error: Command "yarn run build" exited with 1


```

**Comment by tmm** [comment]
What version of wagmi are you using? Are you sure it's installed and the lockfile is up-to-date?

**Comment by emmajane1313** [comment]
using  "wagmi": "^0.7.9" and yes it's definitely installed, did a full uninstall and reinstall too just to be safe and error is persisting

**Reply by tmm** [reply]
Haven't been able to reproduce. Can you put together a [minimal reproducible example](
https://stackoverflow.com/help/minimal-reproducible-example)? I would also try clearing the [Vercel build cache](https://github.com/vercel/community/discussions/457).

**Reply by emmajane1313** [reply]
here is the repo: https://github.com/F3Manifesto/homeport-website/tree/main

i got it working by creating a decs.d.ts file and declared wagmi, ethers and rainbow kit modules
